README.md

```diff
@@ -1,4 +1,9 @@
-**Only UNIX-like systems and public repos at the moment**
+**Watermelon only works with public repos at the moment**
+
+![Discord](https://img.shields.io/discord/933846506438541492?style=flat-square)
+
+[![Stargazers repo roster for @watermelontools/wm-extension](https://reporoster.com/stars/watermelontools/wm-extension)](https://github.com/watermelontools/wm-extension/stargazers)
```

Comment:
As per the [vscode docs](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions) this _might_ not work unless we are a validated publisher